src/diffpy/pdfgui/gui/phaseconfigurepanel.py

```diff
@@ -319,7 +319,7 @@ def restrictConstrainedParameters(self):
         """Set 'read-only' boxes that correspond to constrained parameters."""
 
         self.setToolTips(tooltips.phasepanel)
-        txtbg = self.textCtrlA.DefaultStyle.BackgroundColour
+        # txtbg = self.textCtrlA.DefaultStyle.BackgroundColour
```

Comment:
This change will change behavior. If flake8 is complaining about txtbg not being used, just remove the `txtbg =` from the beginning of the line.